ui/app/src/components/shared/notifications/NotificationPanel.tsx

```diff
@@ -71,6 +113,14 @@ export const NotificationPanel: React.FunctionComponent = () => {
     updatedNotifications.splice(i, 1, n);
     setNotifications(updatedNotifications);
     if (completedStates.includes(n.operation.status) && !isOpen) setShowCallout(true);
+
+    // basic implementation for now, but this can be expanded to give more detailed info to subscribing components on operation changes
+    opsContext.addResourceUpdate({
+      resourceId: n.resource.id,
+      operation: n.operation,
+      componentAction: completedStates.includes(n.operation.status) ?
+        (n.operation.status === "deleted" ? ComponentAction.Remove : ComponentAction.Reload) : ComponentAction.Lock
```

Comment:
`nit` split out the ternary to a new line so its more obvious what condition is nested, i.e. 
```typescript
componentAction: completedStates.includes(n.operation.status)
    ? (n.operation.status === "deleted" ? ComponentAction.Remove : ComponentAction.Reload)
    : ComponentAction.Lock
```